miso-web/src/main/webapp/pages/editBox.jsp

```diff
@@ -281,16 +281,22 @@
   });
 
   jQuery('#selectedBarcode').keyup(function(event) {
-    if(event.keyCode == 13){
-      Box.lookupBoxableByBarcode();
+    if(event.which == "13"){
+      jQuery('#lookupBarcode').click();
     }
   });
 
   jQuery('#selectedBarcode').on('paste', function(e) {
     window.setTimeout(function() {
-      Box.lookupBoxableByBarcode();
+      jQuery('#lookupBarcode').click();
     }, 100);
   });
+
+  jQuery('#updateSelected').keyup(function(event) {
```

Comment:
Is there a case where hitting Enter on a button doesn't already fire a click event?